configs/BROTHERHOBBYF722V3/config.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+#define BOARD_NAME        BHF722V3
+#define MANUFACTURER_ID   BROT
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_MS5611
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25N02K
+#define USE_MAX7456
+
+#define BEEPER_PIN           PC3
+#define MOTOR1_PIN           PB0
+#define MOTOR2_PIN           PB1
+#define MOTOR3_PIN           PB10
+#define MOTOR4_PIN           PB11

```

Comment:
```
#define MOTOR3_PIN           PB10
#define MOTOR4_PIN           PB11
```
These pins are missing timer and DMA assignments.

Reply:
> ```
> #define MOTOR3_PIN           PB10
> #define MOTOR4_PIN           PB11
> ```
> 
> These pins are missing timer and DMA assignments.

I've added timers and DMA allocations to them, please check for the latest committed changes. If you have any questions, please feel free to contact us.